docs/rp2/quickref.rst

```diff
@@ -84,19 +97,36 @@ Use the :ref:`machine.Pin <machine.Pin>` class::
 UART (serial bus)
 -----------------
 
+There are two UARTs, UART0 and UART1. UART0 can be mapped to GPIO 1/2, 12/13
+and 16/17, and UART1 to GPIO 6/7 and 8/9. Since for using the UART GPIO
+numbers have to be used, these should be mentioned and can be seen in the pin
+diagram below:
+
+.. image:: img/UARTpinout.png
```

Comment:
This image has not been added as part of this PR, so the docs have an error when building.

What I would suggest is to change the image at the top of the quickref to the full Pico pinout, eg the one here: https://peppe8o.com/raspberry-pi-pico-pinout/